scripts/workflow/update_wishevent.ts

```diff
@@ -71,10 +71,14 @@ const parseContentBanners = (html: string): ContentBanner[] => {
 						type = 'Weapon';
 					} else if (
 						duration === 'Indefinite' ||
-						name.includes('Wanderlust Invocation') ||
-						name === "Beginners' Wish"
+						name.includes('Wanderlust Invocation')
 					) {
 						type = 'Permanent';
+					} else if (
+						duration === 'Indefinite' ||
+						name === "Beginners' Wish"
+					) {
+						type = "Beginner";
 					}
```

Comment:
`|| `means `duration === 'Indefinite` will always be assigned
you can remove `duration === 'Indefinite` and just use the includes and move your if above
```
else if(name === "Beginners' Wish"){} // will be checked first
else if(duration === 'Indefinite' || name.includes('Wanderlust Invocation') {}
```